File: practise/python/p5.py
# print odd number rang 0 to 100 find sum 
sum_odd = 0
print("Odd numbers from 0 to 100 are:")

for i in range(1, 101, 2):
    print(i, end=" ")
    sum_odd += i

print("\nSum of odd numbers =", sum_odd)

File: practise/c/p5.c
// print odd number rang 0 to 100 find sum 
#include <stdio.h>

int main() {
    int i, sum = 0;

    printf("Odd numbers from 0 to 100 are:\n");
    for (i = 1; i <= 100; i += 2) {
        printf("%d ", i);
        sum += i;
    }

    printf("\nSum of odd numbers = %d\n", sum);
    return 0;
}
